include/bus.hpp

```diff
@@ -59,8 +59,8 @@ class BusType {
 
   // Is there a value available that should be send to a client?
   bool read(data& d);
-  size_t write(uint8_t symbol);
-  int availableForWrite();
+  static size_t write(uint8_t symbol);
+  static int availableForWrite();
```

Comment:
are you sure about these?
moreover, the implementation is not defined static.

Reply:
These are just suggestions from cppecheck - we can leave it as it is

Reply:
these two do not really make sense to me, the rest is fine